Lombiq.Tests.UI.Samples/Tests/RemoteTests.cs

```diff
@@ -13,13 +12,26 @@ namespace Lombiq.Tests.UI.Samples.Tests;
 // to the internet.
 
 // However, sometimes in addition to this, you also want to test remote apps available online, like running rudimentary
-// smoke tests on your production app (e.g.: Can people still log in? Are payments still working?). The UI Testing
+// smoke tests on your production app (e.g.: Can people still log in? Are payments still working?). You can also run
+// such tests in your pre-production environment (like staging) before rolling it out the production. The UI Testing
 // Toolbox also supports this. Check out the example below!
 
 // Note how the test derives from RemoteUITestBase this time, not UITestBase. This is a generic base class for any
 // remote test. However, if you're testing an app behind Cloudflare, you might see requests rejected with an HTTP 403
 // due to Cloudflare's Bot Fight Mode; in that case, derive from CloudflareRemoteUITestBase instead after checking out
 // its documentation on how to set it up.
+
+// While there may be some common code between local and remote tests, they cannot be the same, and the common code must
+// be in a project independent of the web app (which is not the case here for the sake of simplicity):
+// - In local tests, you have a much lower-level access to the app: E.g., you can access the Orchard Core log, use
```

Comment:
Did you mean "higher-level"?

```suggestion
// - In local tests, you have much higher-level access to the app: E.g., you can access the Orchard Core log, use
```

Reply:
I did mean lower level: the app runs in the same process, so you can communicate with its container and types directly, whereas in prod, all you have is the public UI.